vsintegration/Vsix/VisualFSharpFull/VisualFSharpFull.csproj

```diff
@@ -48,7 +48,6 @@
       <NgenArchitecture>All</NgenArchitecture>
       <NgenPriority>2</NgenPriority>
       <Private>True</Private>
-      <AdditionalProperties>TargetFramework=$(DependencyTargetFramework)</AdditionalProperties>
```

Comment:
Will you check that the assemblies that are getting included in VisualFSharpFull.vsix are the correct TFM?  As I recall, these lines were necessary to avoid double-writes and potentially getting the wrong TFM, e.g., a copy of FSharp.Compiler.Private for netstandard that can't write Windows PDBs.